python/ml4ir/tests/test_ranking_model.py

```diff
@@ -64,8 +65,8 @@ def test_csv_and_tfrecord(self):
         )
 
         # Check if the loss and accuracy on the test set is the same
-        assert np.isclose(csv_loss, 0.62368, rtol=0.05)
-        assert np.isclose(csv_mrr, 0.564156, rtol=0.05)
+        assert np.isclose(csv_loss, 0.55835, rtol=0.05)
```

Comment:
these magic numbers... what are we testing here?  Every time you add something new, they'll change.  How will you know if the change is good?

Reply:
You are right. I'm not sure. do you have any ideas for doing an end-to-end test?